src/App.js

```diff
@@ -1,26 +1,31 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import { Switch, Route, Link } from 'react-router-dom';
+import FooterTabs from './components/FooterTabs';
```

Comment:
No need to change this...we can clean it up later. This `Link` isn't used here, so we can get rid of it. Next time, tho!